_topic_map.yml

```diff
@@ -122,6 +122,8 @@ Name: Installing
 Dir: installing
 Distros: openshift-origin,openshift-enterprise,openshift-webscale
 Topics:
+- Name: Installation overview
+  File: ocp-installation-overview
```

Comment:
@pneedle-rh rename this to index. This will allow us to have a landing page for installation docs like: 

https://docs.openshift.com/container-platform/4.6/installing/ or
https://docs.openshift.com/container-platform/latest/installing/ 

and it will always redirect to this page.

Reply:
Thank you for this suggestion. This has now been applied.